### DISCUSSION TITLE
Welcome to fabric-debugger Discussions!

### DISCUSSION THREAD

**Discussion by ashwath-spdr** [spydra-tech/fabric-debugger#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the community to bounce ideas around Hyperledger Fabric Debugger. We hope that you:
  * Ask any queries about the Hyperledger Fabric Debugger.
  * Share ideas, enhancements and feature requests.
  * Engage with other community members.

**Comment by demianko** [comment]
it works geat with chaincode in go.

do you have any plan to support chaincode in java in near future?

**Reply by ashwath-spdr** [reply]
Hi @demianko, 
yes, this is being tracked as part of feature request #2. This should be released in a weeks time.
